core/src/main/java/com/dtolabs/rundeck/core/execution/ExecutionServiceImpl.java

```diff
@@ -384,7 +384,9 @@ public String fileCopyScriptContent(ExecutionContext context, String script, INo
 
     public NodeExecutorResult executeCommand(final ExecutionContext context, final String[] command,
                                              final INodeEntry node) {
-        return executeCommand(context, ExecArgList.fromStrings(DataContextUtils
+
+        boolean featureQuotingBackwardCompatible = Boolean.valueOf(context.getIFramework().getPropertyRetriever().getProperty("rundeck.feature.quoting.compatibility"));
```

Comment:
Can we change the configuration variable name to `rundeck.feature.quoting.backwardCompatible` to make it more consistent with the parameter name?
Thanks!